src/scripts/opm_status.js

```diff
@@ -38,12 +37,10 @@ module.exports = (robot) => {
       incrementStats("OPM status");
 
       try {
-        const { data, status } = await axios.get(
+        const data = await fetch(
           "https://www.opm.gov/json/operatingstatus.json",
-        );
-        if (status !== 200) {
```

Comment:
[question] can we get rid of this error throw because `data` is more consistent now?

Reply:
Yup